src/rgw/driver/sfs/CHANGELOG.md

```diff
@@ -7,7 +7,23 @@ The format is based on
 and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.11.0] - Unreleased
```

Comment:
let's drop this change, and add the release notes to the wiki at https://github.com/aquarist-labs/s3gw/wiki/Current-CHANGELOG